src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -1152,16 +1174,24 @@ private void processReferenceData(Map<Long, ExtractCohortRecord> currentPosition
     }
 
     private ExtractCohortRecord processReferenceData(Iterator<GenericRecord> sortedReferenceRangeIterator, Map<Long, TreeSet<ReferenceRecord>> referenceCache, long location, long sampleId) {
-        String state = processReferenceDataFromCache(referenceCache, location, sampleId);
+        ReferenceRecord r = processReferenceDataFromCache(referenceCache, location, sampleId);
 
-        if (state == null) {
-            state = processReferenceDataFromStream(sortedReferenceRangeIterator, referenceCache, location, sampleId);
+        if (r == null) {
+            r = processReferenceDataFromStream(sortedReferenceRangeIterator, referenceCache, location, sampleId);
         }
 
-        return new ExtractCohortRecord(location, sampleId, state);
+        return new ExtractCohortRecord(location, sampleId, r.getState());
     }
 
-    private String processReferenceDataFromCache(Map<Long, TreeSet<ReferenceRecord>> referenceCache, long location, long sampleId) {
+    // TODO: use this as a return record, singleton
+    // TODO: so much hackery to instantiate this thing...
+    public class InferredReferenceRecord extends ReferenceRecord {
+        public InferredReferenceRecord(GQStateEnum inferredState) {
+            super(SchemaUtils.chromAdjustment + 1, -1, -1, inferredState.getValue());
```

Comment:
What is the reason for using `chromAdjustment` here? Are you just arbitrarily choosing Chr1:1 here?

Reply:
Yep -- it's arbitrary but required by the superclass (even though we don't need it here).  I improved the comments to reflect this instead of what I had before about "so much hackery" :D